command/format/diff.go

```diff
@@ -361,6 +374,10 @@ func (p *blockBodyDiffPrinter) writeAttrDiff(name string, attrS *configschema.At
 	return false
 }
 
+func (p *blockBodyDiffPrinter) writeNestedAttrDiff(name string, attrS *configschema.Attribute, old, new cty.Value, nameLen, indent int, path cty.Path) bool {
+	panic("not implemented")
+}
```

Comment:
This uncalled method confused me a bit, and I couldn't quite see where you'd go next with it. Would it be possible to add a short FIXME comment explaining where this would be called from and some of the next steps?

Reply:
updated! the new comment should make it easier to link the TBD new function, where it's going to be called inside `writeAttrDiff`, and the tests with FIXME comments that show the current, less desirable, output.

Reply:
Thank you! 🙇